docs/content/preview/yugabyte-voyager/reference/data-migration/export-data.md

```diff
@@ -8,65 +8,83 @@ menu:
-
-- [export data](#export-data)
+- [export data](#export-data) (and export data from source)
 - [export data status](#export-data-status)
+- [get data-migration-report](#get-data-migration-report)
+- [export data from target](#export-data-from-target)
 
 ## export data
 
 For offline migration, export data [dumps](../../../migrate/migrate-steps/#export-data) data of the source database in the `export-dir/data` directory on the machine where yb-voyager is running.
 
-For [live migration](../../../migrate/live-migrate/#export-data) (and [fall-forward](../../../migrate/live-fall-forward/#export-data)), export data [dumps](../../../migrate/live-migrate/#export-data) the snapshot in the `data` directory and starts capturing the new changes made to the source database.
+For [live migration](../../../migrate/live-migrate/#export-data) (with [fall-forward](../../../migrate/live-fall-forward/#export-data) and [fall-back](../../../migrate/live-fall-back/#export-data)), the export data command is an alias of `export data from source` which [dumps](../../../migrate/live-migrate/#export-data) the snapshot in the `data` directory and starts capturing the new changes made to the source database.
 
 ### Syntax
 
+Syntax for export data is as follows:
+
 ```text
 Usage: yb-voyager export data [ <arguments> ... ]
 ```
 
-#### Arguments
+Syntax for export data from source is as follows:
+
+```text
+Usage: yb-voyager export data from source [ <arguments> ... ]
+```
+
+### Arguments
 
 The valid *arguments* for export data are described in the following table:
 
 | Argument | Description/valid options |
 | :------- | :------------------------ |
-| --disable-pb | Use this argument to not display progress bars. For live migration, `--disable-pb` can also be used to hide metrics for export data. (default: false) |
-| --table-list | Comma-separated list of the tables for which data needs to be migrated. Do not use in conjunction with `--exclude-table-list`. |
-| --exclude-table-list <tableNames> | Comma-separated list of tables to exclude while exporting data. For export data command, the list of table names passed in the `--table-list` and `--exclude-table-list` are, by default, case insensitive. Enclose each name in double quotes to make it case sensitive.|
+| --disable-pb |Use this argument to disable progress bar during data export and statistics printing during streaming phase. <br>Default: false<br> Accepted parameters: true, false, yes, no, 0, 1 |
 | -e, --export-dir <path> | Path to the export directory. This directory is a workspace used to store exported schema DDL files, export data files, migration state, and a log file.|
-| --export-type | Choose migration type between `snapshot-only` (offline) or `snapshot-and-changes`(live, and optionally fall-forward). (default: `snapshot-only`) |
+| --export-type | Specify the migration type as `snapshot-only` (offline) or `snapshot-and-changes` (live, and optionally fall-forward). <br>Default: `snapshot-only` |
```

Comment:
optionally fall forward? you don't need to specify 'snapshot-and-changes' if you want to do fall forward, or fall back for that matter? not clear

Reply:
export-type is snapshot-and-changes for live migration for all types. So ill change it. Thanks